src/commands/SuggestedActionsCard.ts

```diff
@@ -61,6 +61,16 @@ async function processor(context: TurnContext, arg: string) {
             value: {
               hello: 'World!'
             },
+          },
+          {
+            displayText: 'Aloha',
+            image: `${ PUBLIC_URL }assets/square-icon-purple.png`,
+            text: 'echo Hello',
+            title: 'Message back as string with display text',
+            type: 'messageBack',
+
+            // TODO: Remove value after bumping DLJS
```

Comment:
Maybe wait for a few minutes and bump DLJS so we can remove this.

Reply:
sounds good